documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -6,16 +6,28 @@ Hot Rod is a binary TCP protocol that {brandname} provides for high-performance
 
 .Client intelligence
 
-The Hot Rod protocol includes a mechanism that provides clients with an up to date view of the cache topology.
+The Hot Rod protocol includes a mechanism that provides clients with an up-to-date view of the cache topology.
 Client intelligence improves performance by reducing the number of network hops for read and write operations.
 
+`HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism. `HASH_DISTRIBUTION_AWARE` is the best mechanism for allowing clients to route requests to primary owners, which improves a client's performance.
+
+[NOTE]
+====
+Hot Rod clients must use `BASIC` intelligence in the following situations:
+
+* Connecting to {brandname} through a `LoadBalancer`, `NodePort`, or {openshiftshort} `Route`.
+* Running a client in one {k8s} cluster and the client wants to access a different {k8s} cluster when cross-site failover is required.
```

Comment:
```suggestion
* Accessing a different {k8s} cluster when using cross-site replication.
```

Reply:
Should always use "cross-site replication" not "cross-site failover" for clarity and consistency. Cross-site replication is the actual name of the capability.

Reply:
In this case we're not talking about the capability, we're talking about the act of the client failing over from one k8s cluster to another.

Reply:
Thanks, @oraNod . I've added this terminology to my notes.

Reply:
Oh, sorry @ryanemerson . Revery to `Running a client in one {k8s} cluster and the client wants to access a different {k8s} cluster when cross-site failover is required.` ?

Reply:
How about "Failing over to a different {k8s} cluster when using cross-site replication."